src/gt4sd/training_pipelines/moses/organ/core.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import logging
+from dataclasses import dataclass, field
+from typing import Any, Dict
+
+from guacamol_baselines.moses_baselines.organ_train import main
+from moses.script_utils import MetricsReward
+
+from ...core import TrainingPipelineArguments
+from ..core import MosesTrainingPipeline
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
+
+
+class MosesOrganTrainingPipeline(MosesTrainingPipeline):
+    """Moses Organ training pipelines."""
+
+    def train(  # type: ignore
+        self,
+        model_args: Dict[str, Any],
+        common_args: Dict[str, Any],
+    ) -> None:  # type: ignore
+        params = {**common_args, **model_args}
+        parser = argparse.ArgumentParser()
+        for k, v in params.items():
+            parser.add_argument("--" + k, default=v)
+        args = parser.parse_known_args()[0]
+        print(vars(args))
```

Comment:
remove print

Reply:
I have pushed a change for this, thanks :)